src/Tribe/Attendee_Registration/View.php

```diff
@@ -37,138 +25,14 @@ public function display_attendee_registration_page() {
 	 * @return string The resulting template content
 	 */
 	public function display_attendee_registration_shortcode() {
-		$q_provider = tribe_get_request_var( 'provider', false );
-
-		/**
-		 * Filter to add/remove tickets from the global cart
-		 *
-		 * @since 4.9
-		 * @since 4.11.0 Added $q_provider to allow context of current provider.
-		 *
-		 * @param array  $tickets_in_cart The array containing the cart elements. Format array( 'ticket_id' => 'quantity' ).
-		 * @param string $q_provider      Current ticket provider.
-		 */
-		$tickets_in_cart = apply_filters( 'tribe_tickets_tickets_in_cart', [], $q_provider );
-
-		$events           = [];
-		$providers        = [];
-		$default_provider = [];
-
-		foreach ( $tickets_in_cart as $ticket_id => $quantity ) {
-			// Load the tickets in cart for each event, with their ID, quantity and provider.
-
-			/** @var Tribe__Tickets__Tickets_Handler $handler */
-			$handler = tribe( 'tickets.handler' );
-			$ticket  = $handler->get_object_connections( $ticket_id );
-
-			if ( ! $ticket->provider instanceof Tribe__Tickets__Tickets ) {
-				continue;
-			}
-
-			// Check for attendee object or class name.
-			$ticket_providers = [
-				$ticket->provider->attendee_object,
-				get_class( $ticket->provider ),
-			];
-
-			if ( ! empty( $ticket->provider->orm_provider ) ) {
-				$ticket_providers[] = $ticket->provider->orm_provider;
-			}
-
-			// If we've got a provider and it doesn't match, skip the ticket
-			if ( ! in_array( $q_provider, $ticket_providers, true ) ) {
-				continue;
-			}
-
-			$ticket_data = [
-				'id'       => $ticket_id,
-				'qty'      => $quantity,
-				'provider' => $ticket->provider,
-			];
-
-			/**
-			 * Flag for event form to flag TPP. This is used for the AJAX
-			 * feature for save attendee information. If the provider is
-			 * TPP, then AJAX saving is disabled.
-			 *
-			 * @todo: This is temporary until we can figure out what to do
-			 *        with the Attendee Registration page handling multiple
-			 *        payment providers.
-			 */
-			$provider = '';
-
-			if ( empty( $default_provider ) ) {
-				// One provider per instance.
-				$default_provider[ $q_provider ] = $ticket->provider->class_name;
-			}
-
-			/** @var Tribe__Tickets__Status__Manager $status */
-			$status   = tribe( 'tickets.status' );
-			$provider = $status->get_provider_slug( $ticket->provider->class_name );
+		try {
+			/* @var Tribe__Tickets_Plus__Attendee_Registration__View $view */
```

Comment:
:warning: This comment is 50% valid code; is this commented out code?

<sub>hash: 30774923fc13b6466fa167958f4575b3</sub>